Gemfile

```diff
@@ -2,10 +2,18 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'rake', '~> 10.4', '>= 10.4.2'
+
+group :test do
+  gem 'rspec', '~> 3.0'
+  gem 'rspec-its', '~> 1.1'
```

Comment:
Moving these gems out of the `development` group in the gemspec and into their own test group allows non-test development gems to be excluded from the CI install, which speeds up the build. The only disadvantage is that `gem install addressable --development` doesn’t install all the development dependencies, but it wasn’t doing that before anyway. Working on Addressable in development requires Bundler, so I’ve made Bundle the sole development dependency, which gives us finer-grained control in the `Gemfile`. `Gem::Specification` only allows coarse-grained, development or runtime dependencies.